tracer/src/Datadog.Trace/IAST/Aspects/StringAspect.cs

```diff
@@ -0,0 +1,244 @@
+// <copyright file="StringAspect.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using Datadog.Trace.Iast.Dataflow;
```

Comment:
use #nullable enable in new files

Reply:
Not for aspect classes. This would change the function definition and could break the native processor